packages/webawesome/docs/_includes/base.njk

```diff
@@ -48,9 +41,14 @@
       }
     </script>
   </head>
-  <body class="layout-{{ layout | stripExtension }} page-{{ page.fileSlug or 'home' }}{{ ' page-wide' if wide }} page-{{ pageType or 'docs' }}">
+  <body class="layout-{{ layout | stripExtension }} page-{{ page.fileSlug or 'home' }}{{ ' page-wide' if wide }}">
     <!-- use view="desktop" as default to reduce layout jank on desktop site. -->
-    <wa-page view="desktop" disable-navigation-toggle {% if pageType == 'marketing' %}disable-sticky="header"{% endif %} mobile-breakpoint="1180">
+    <wa-page
+      view="desktop"
+      {% set waPage = waPage or {} %}
+      {% if waPage.disableNavigationToggle is not defined or waPage.disableNavigationToggle %}disable-navigation-toggle{% endif %}
+      mobile-breakpoint="{{ waPage.mobileBreakpoint or 1180 }}"
```

Comment:
@KonnorRogers this works for overriding already predefined page props, but in our site we also want to add properties like `disable-sticky`. Is there a better way to do this?

Reply:
We paired on this and figured out a better way to handle this in 3a3f4e9a56f04ddc62703dfd1ce811df3257dcec.